index.rst

```diff
@@ -1,61 +1,338 @@
+
+IDM-1203
+    Quota grants may expire.
+    The user must be notified via email of pending quota grant expirations.
+
+Administration
+--------------
+
+IDM-1300
+    Administrators of the Science Platform must be able to modify any of the user's metadata on behalf of the user.
+
+IDM-1301
+    Administrators of the Science Platform must be able to set and change expiration dates on accounts.
+
+IDM-1302
+    Administrators of the Science Platform must be able to approve a pending change of email address even if the user has not responded to the challenge.
+
+IDM-1303
+    Administrators of the Science Platform must be able to create, revoke, and change the expiration dates on quota grants.
+
+IDM-1304
+    Administrators must be able to impersonate a user and see the same thing that a user would see in the user metadata interface.
+
+Logging
+-------
+
+IDM-1400
+    All changes to account metadata must be logged.
+    If the changes were made by an administrator instead of the user, this must be clearly indicated in the logs.
+
+IDM-1401
+    All changes to quotas associated with users must be logged.
+
+IDM-1402
+    Users must be able to see a history of all of their quota changes.
+
+Groups
+======
+
+Management
+----------
+
+IDM-2000
+    Users may be members of zero or more groups.
+
+IDM-2001
+    Groups can be configured to control membership based on attributes provided by the identity provider.
+    Membership in those groups must be tied to affiliation information from specific identity providers and dynamically adjusted if an authentication from that identity provider stops returning the same metadata.
+    Multiple identity providers may provide access to the same group.
+    In this case, the membership should only be withdrawn if all those identity providers stop providing the relevant information.
+    It must be possible to periodically force users to authenticate with an attribute-providing identity provider to reconfirm access to their groups, similar to IDM-0009.
+
+IDM-2002
+    Users must be able to create their own groups.
+    The owner of the group must then be able to add and remove members as they wish.
+    Owners must also be able to add additional group owners who can then also control membership in the group.
+
+IDM-2003
+    It must be possible to create groups whose membership can only be changed by Science Platform administrators.
+
+IDM-2004
+    Groups must be checked against namespace rules that, for instance, force all groups created by a user to start with a specific prefix that includes their username.
+
+IDM-2005
+    Group membership may include an expiration date.
+    When the expiration date is reached, the user will be automatically removed from the group.
+    Anyone who can control membership in the group must be able to update the expiration date.
+
+IDM-2006
+    Owners must be able to rename groups while preserving all quota grants and membership.
```

Comment:
This doesn't say it but it implies that each group will have some kind of gid or id, and that the name shouldn't be used  (kind of like how you said for users)

Reply:
Yup, good call, added that explicitly.